### PR TITLE
[17.0][MIG] contract_sale: Migration to 17.0

### PR DESCRIPTION
**Pull request by jaydeep32**

### CODE REVIEW

**Comment by jaydeep32** [comment]
Hello @CRogos , this one is a new PR based on #1049.
I hope everything is good now..

File: contract_sale/__manifest__.py [hunk 1]
```diff
@@ -12,7 +12,10 @@
     "data": [
         "security/ir.model.access.csv",
         "security/contract_security.xml",
+        "views/abstract_contract_line.xml",
         "views/contract.xml",
+        "views/contract_line.xml",
+        "views/contract_template.xml",
         "views/res_partner_view.xml",
```
Comment (CRogos) [inline contract_sale/__manifest__.py]: Is this contract_sale related?

**Review by CRogos — COMMENTED** [review]
Yes, looks a lot better now.
You could squash the marked commits according to:
https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate

or in short:
git config --global --add rebase.instructionFormat "(%an <%ae>) %s"
git rebase -i origin/17.0
than replace "pick" with "f" where needed and close the file.
git push -f

image

**Review by CRogos — APPROVED** [review]
Code + Function LGTM

@MohamedOsman7 could you review?

**Comment by CRogos** [comment]
Same changes as https://github.com/OCA/contract/pull/1049 but clean commit history.

**Review by MohamedOsman7 — APPROVED** [review]
Code & Functional review LGTM

**Comment by CRogos** [comment]
@rousseldenis could you merge?
